topics/ingest-feedbacks.yaml

```diff
@@ -0,0 +1,17 @@
+topic: ingest-feedbacks
+pipeline: replays
+description: User feedback widget events
+services:
+  producers:
+    - getsentry/sentry
+  consumers:
+    - getsentry/sentry
+schemas:
+  - version: 1
+    compatibility_mode: none
+    type: json
+    resource: events.v1.schema.json
+    examples:
+      - events/1/
+topic_creation_config:
+  message.timestamp.type: LogAppendTime
```

Comment:
I don't think you need this, you can remove the whole topic_creation_config block unless you have some special configuration you want to enforce